jumble/src/utils/credentials.ts

```
@@ -41,10 +45,23 @@ export function getPublicKeyCredentialDescriptor(
   storedCredential: StoredCredential | null,
 ): PublicKeyCredentialDescriptor | undefined {
   if (storedCredential?.method === "passkey") {
-    return {
-      id: Uint8Array.from(atob(storedCredential.id), (c) => c.charCodeAt(0)),
-      type: "public-key" as PublicKeyCredentialType,
-    };
+    try {
+      // Check if the string is valid base64
+      if (!isValidBase64(storedCredential.id)) {
+        console.warn("Invalid base64 format in stored credential ID");
+        return undefined;
```

I don't know if silently eating errors is a good idea?